abelfunctions/riemann_surface_path.py

```diff
@@ -335,7 +335,7 @@ def get_y(self, s):
         yi = self.analytically_continue(xim1, yim1, xi)
         return yi
 
-    def analytically_continue(xi, yi, xip1):
+    def analytically_continue(self, yi, xip1):
```

Comment:
The method should still be a function of xi
```suggestion
    def analytically_continue(self, xi, yi, xip1):
```